.nvmrc

```diff
@@ -1 +1 @@
-v14.16.1
\ No newline at end of file
+v16.15.1
```

Comment:
Something I've wondered about recently, should we prefer just specifying the major version - e.g. `v16`? I'm not sure of the pros/cons here vs being more explicit.